dialoguekit/core/dialogue_act.py

```diff
@@ -0,0 +1,17 @@
+"""Interface representing a dialogue act."""
```

Comment:
Could you expand the description with two specific examples of dialogue acts? E.g., one with and one without annotations.